datus/storage/backends/relational/sqlalchemy_backend.py

```
@@ -0,0 +1,628 @@
+        return self
+
+    def __exit__(self, exc_type: Any, exc_val: Any, exc_tb: Any) -> None:
+        if exc_type is not None:
+            self.rollback()
+        elif not self._committed and not self._rolled_back:
+            self.commit()
+
+        if self._conn is not None:
+            try:
+                self._conn.close()
+            except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)